config.json.orig

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Should not it be removed?

Reply:
Wanted to leave it as it shows some other options for Mermaid CSS.